recipes-kernel/linux/linux-fslc-imx_5.15.87/0006-arm64-dts-iesy-iesy-imx8mm-eva-mi-add-SPI-B.patch

```diff
@@ -0,0 +1,34 @@
+From 4eb2b6ecbf66f6dc238dd99e620065e2ff21b331 Mon Sep 17 00:00:00 2001
+From: Dominik Poggel <pog@iesy.com>
+Date: Fri, 20 Oct 2023 12:03:48 +0200
+Subject: [PATCH 6/6] arm64: dts: iesy: iesy-imx8mm-eva-mi: add SPI B
+
+---
+ arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi | 11 +++++++++++
+ 1 file changed, 11 insertions(+)
+
+diff --git a/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi b/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
+index 4ef12b9f0cdf..ff03b0a0838d 100644
+--- a/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
++++ b/arch/arm64/boot/dts/iesy/iesy-imx8mm-osm-sf.dtsi
+@@ -177,6 +177,17 @@ &wdog1 {
+ 	status = "okay";
+ };
+ 
++/* SPI B */
++&ecspi1 {
++	#address-cells = <1>;
++	#size-cells = <0>;
++	fsl,spi-num-chipselects = <1>;
++	pinctrl-names = "default";
++	pinctrl-0 = <&pinctrl_ecspi1 &pinctrl_ecspi1_cs>;
++	cs-gpios = <&gpio5 9 GPIO_ACTIVE_LOW>;
++	status = "disabled";
```

Comment:
Why is the status "disabled"?

Reply:
The interface is implemented in the modules dtsi file, but disabled, as it is not in use by the module itself. It is enabled by the boards dts file. Prevents that unused interfaces are probed, when the module is included later on in another custom board file. 
That in mind, rework of the dtsi file is needed for the other interfaces as well